Sources/NIOSSH/SSHMessages.swift

```diff
@@ -35,7 +35,7 @@ enum SSHMessage: Equatable {
     case newKeys
     case userAuthRequest(UserAuthRequestMessage)
     case userAuthFailure(UserAuthFailureMessage)
-    case userAuthSuccess
+    case userAuthSuccess(String?)
```

Comment:
We don't want to make this change. This enum stores strictly the messages, as they are represented on the wire. This message does not carry a username, so we shouldn't be attaching it here.

Reply:
Good enough, it felt dirty when I did it. I have another strategy for connecting from the escaped closure world of the authentication delegate to the state structures.